docs/releases/1.10.txt

```diff
@@ -24,7 +24,35 @@ recommend** and only officially support the latest release of each series.
 What's new in Django 1.10
 =========================
 
-...
+Autocomplete Fields
+-------------------
+
+```ModelAdmin`` supports new
+`autocomplete_fields <django.contrib.admin.AdminSite.autocomplete_fields>`
+for the ``ForeignKeyField`` and ``Many2ManyField``.
+They look and work similar to regular select inputs,
+but with an additional search field. Furthermore
+autocomplete fields load choices asynchronous. This makes them
+suitable to handle foreign relations very large result sets.
+
+The autocomplete fields work similar to
```

Comment:
I think one paragraph or so is probably enough. We don't really need to repeat examples from the linked documentation.